lib/data/copy/leaders_controller.yml

```diff
@@ -0,0 +1,10 @@
+welcome: >
```

Comment:
Let's change this to `welcome: |`. 
See http://stackoverflow.com/a/21699210/4018167.